### PR TITLE
 Add First SDK for LabServices RP 2018-10-15

### PR DESCRIPTION
**Pull request by ianhays**
## Description

---

This checklist is used to make sure that common guidelines for a pull request are followed.
- [X] Please add REST spec PR link to the SDK PR
  - https://github.com/Azure/azure-rest-api-specs/pull/4621
- [X] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [X] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [X] Title of the pull request is clear and informative.
- [X] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [x] Pull request includes test coverage for the included changes.
  - https://github.com/Azure/azure-rest-api-specs/issues/4738

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [x] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
  - https://github.com/Azure/azure-rest-api-specs/pull/4621
- [x] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code.
- [x] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

**Comment by ianhays** [comment]
@shahabhijeet PTAL. Tests were added.

File: src/SDKs/LabServices/LabServices.Tests/Properties/launchSettings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (shahabhijeet) [inline src/SDKs/LabServices/LabServices.Tests/Properties/launchSettings.json]: @ianhays remove this file

**Review by shahabhijeet — CHANGES_REQUESTED** [review]

File: src/SDKs/LabServices/LabServices.Tests/LabServices.Tests.csproj [hunk 2]
```diff
@@ -0,0 +1,31 @@
+﻿<Project Sdk="Microsoft.NET.Sdk" ToolsVersion="15.0">
+  <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.test.reference.props'))" />
+  <PropertyGroup>
+    <Description>LabServices.Tests Class Library</Description>
+    <PackageId>LabServices.Tests</PackageId>
+    <AssemblyName>LabServices.Tests</AssemblyName>
+    <Version>1.0.0</Version>
+    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
```
Comment (shahabhijeet) [inline src/SDKs/LabServices/LabServices.Tests/LabServices.Tests.csproj]: @ianhays for some reason it seems my previous comments were not posted.
Please remove line 8-14

File: src/SDKs/LabServices/LabServices.Tests/LabServices.Tests.csproj [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+    <FileUpgradeFlags>
+    </FileUpgradeFlags>
+    <UpgradeBackupLocation>
+    </UpgradeBackupLocation>
+    <OldToolsVersion>2.0</OldToolsVersion>
+    <ProjectGuid>{DC1024AF-4E20-4E65-9ED8-09CC26385B8B}</ProjectGuid>
+  </PropertyGroup>
+  <ItemGroup>
+    <!--<PackageReference Include="Microsoft.Azure.Management.LabServices" Version="1.2.0" />-->
```
Comment (shahabhijeet) [inline src/SDKs/LabServices/LabServices.Tests/LabServices.Tests.csproj]: remove commented code

File: src/SDKs/LabServices/Management.LabServices/Microsoft.Azure.Management.LabServices.csproj [hunk 4]
```diff
@@ -0,0 +1,22 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+	<!-- Please do not move/edit code below this line -->
+	<Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
+	<!-- Please do not move/edit code below this line -->
+
+	<PropertyGroup>
+		<Version>3.0.0</Version>
```
Comment (shahabhijeet) [inline src/SDKs/LabServices/Management.LabServices/Microsoft.Azure.Management.LabServices.csproj]: Any reason this is 3.0.0 as your first SDK?

**Review by shahabhijeet — CHANGES_REQUESTED** [review]

File: src/SDKs/LabServices/Management.LabServices/Microsoft.Azure.Management.LabServices.csproj (hunk 4 above)
Comment (ianhays) [reply src/SDKs/LabServices/Management.LabServices/Microsoft.Azure.Management.LabServices.csproj]: For consistency with our other product, DevTestLabs

**Review by shahabhijeet — APPROVED** [review]

**Pull request merged**